Content.Server/GameObjects/Components/Sound/FootstepModifierComponent.cs

```diff
@@ -1,38 +1,14 @@
-using Content.Shared.Audio;
-using Robust.Server.GameObjects;
-using Robust.Shared.Audio;
 using Robust.Shared.GameObjects;
-using Robust.Shared.IoC;
-using Robust.Shared.Player;
-using Robust.Shared.Prototypes;
-using Robust.Shared.Random;
-using Robust.Shared.Serialization.Manager.Attributes;
 
 namespace Content.Server.GameObjects.Components.Sound
 {
     /// <summary>
     /// Changes footstep sound
     /// </summary>
     [RegisterComponent]
-    public class FootstepModifierComponent : Component
+    public class FootstepModifierComponent : BaseEmitSoundComponent
```

Comment:
this got also modified, to try and be consistent

Reply:
this one should probably remain a separate thing IMO, it doesnt really fit well into the EmitSound duplication you're removing

Reply:
this way it's a bit more ECS'y, because instead of having login inside the component, we only have some data, which we read in a class which actually has logic, MoverController in this case